portal/views/teacher/teach_new.py

```diff
@@ -489,6 +490,96 @@ def teacher_move_class_new(request, access_code):
 
 @login_required(login_url=reverse_lazy('login_new'))
 @user_passes_test(logged_in_as_teacher, login_url=reverse_lazy('login_new'))
+def teacher_move_students_new(request, access_code):
```

Comment:
Well you can use the import... as functionality :)
ex: 
`from portal.views.teach import function
from portal.views.teach_new import function as function_new` \o/